Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<p>The top line should have the ### at the start (if it's intended as a title)</p><p> </p><p>Also the units as arbitrary at this point. Could you drop that part and bring units up later in the tutorial, or define them with the synthetic spectrum above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>